mac/bg.lproj/Credits.rtf

```diff
@@ -0,0 +1,37 @@
+{\rtf1\ansi\ansicpg1252\cocoartf949\cocoasubrtf430
```

Comment:
this file is garbled, I think you need to use WordPad or something else to edit an RTF file

Reply:
![Untitled](https://user-images.githubusercontent.com/68707872/155845054-20ad1f5f-b8da-423d-8039-31f4b1590684.png)

This is the end result after editing it in Word Pad.

Reply:
![Untitled](https://user-images.githubusercontent.com/68707872/155845101-af775bbb-1fd2-4d69-b31a-8e4d64a304ee.png)

The file looks fine in Libre Office though.

Reply:
I recreated this file on my end, should work alright now.

Reply:
Looks good.